object_manager.go

```diff
@@ -677,3 +673,56 @@ func (objMgr *ObjectManager) GetGridInfo() ([]Grid, error) {
 	err := objMgr.connector.GetObject(gridObj, "", &res)
 	return res, err
 }
+
+// GetZoneAuth returns the authoritatives zones
+func (objMgr *ObjectManager) GetZoneAuth() ([]ZoneAuth, error) {
+	var res []ZoneAuth
+
+	zoneAuth := NewZoneAuth(ZoneAuth{})
+	err := objMgr.connector.GetObject(zoneAuth, "", &res)
+
+	return res, err
+}
+
+// GetZoneDelegated returns the delegated zone
+func (objMgr *ObjectManager) GetZoneDelegated(fqdn string) (*ZoneDelegated, error) {
```

Comment:
can you make this a varidic function so that it can return all the zones  delegated when no fqdn is passed? and return the particular zone if and when fqdn is passed. The second part is already covered